templates/docker-compose.yml

```diff
@@ -101,6 +101,7 @@ services:
       - ./volumes/postgres:/var/lib/postgresql/data:Z
       - ./customPostgresql.conf:/etc/postgresql.conf
     restart: always
+    command: postgres -c config_file=/etc/postgresql.conf
```

Comment:
Any reason we shouldn't just mount the `postgresql.conf` to the expected default `/var/lib/postgresql/data/postgresql.conf` instead of forcing our own?

If possible, I'd prefer to have only one copy of `postgresql.conf` on the VM to help prevent future confusion.

Reply:
It's not detailed in the official docs, but I don't see why we couldn't. Let me do some testing to confirm a fresh DB start wouldn't try to overwrite it, and I'll update the PR accordingly.

Moving this weekend, so might not be until Monday.

Reply:
And now we know why you don''t mount it there:

```
initdb: error: directory "/var/lib/postgresql/data" exists but is not empty
initdb: hint: If you want to create a new database system, either remove or empty the directory "/var/lib/postgresql/data" or run initdb with an argument other than "/var/lib/postgresql/data".
```

Postgres won't create the proper file system in `/var/lib/postgresql/data` because a file exists in the directory already.

One option would be to use the recommended `/etc/postgresql/postgresql.conf`, but honestly, even with this way it leaves the door open to an administrator using a default config to start and not realizing it won't override at a future time, and in this particular situation it would still result in two files. I would suggest merging as-is to ensure the "use of wrong config" part of this doesn't happen, but let me know if you'd like me to make a change.

Reply:
What a funny chicken-and-egg situation. Thanks for checking. I concur. The path is moot at that point, so we may as well keep it as-is.